modules/wechat_qrcode/src/wechat_qrcode.cpp

```diff
@@ -132,9 +133,20 @@ vector<string> WeChatQRCode::Impl::decode(const Mat& img, vector<Mat>& candidate
                 super_resolution_model_->processImageScale(cropped_img, cur_scale, use_nn_sr_);
             string result;
             DecoderMgr decodemgr;
-            auto ret = decodemgr.decodeImage(scaled_img, use_nn_detector_, result);
-
+            vector<Point2f> points_qr;
+            auto ret = decodemgr.decodeImage(scaled_img, use_nn_detector_, result, points_qr);
             if (ret == 0) {
+                for (int i = 0; i < 4; i++) {
+                    points_qr[i].x /= cur_scale;
+                    points_qr[i].y /= cur_scale;
```

Comment:
It could be just ` points_qr[i] /= cur_scale`

Reply:
fixed